include/qb/qblog.h

```diff
@@ -584,6 +584,7 @@ enum qb_log_conf {
 	QB_LOG_CONF_EXTENDED,
 	QB_LOG_CONF_IDENT,
 	QB_LOG_CONF_MAX_LINE_LEN,
+	QB_LOG_CONF_ELIPSIS,
```

Comment:
fyi spelling is "ellipsis"

Reply:
Thanks, I knew it looked wrong, but couldn't quite figure out why!